### PR TITLE
Polish dist

### PR DESCRIPTION
**Pull request by AWBroscius**
Address post pull request task list todos in [Issue 125](https://github.com/theochem/DiverseSelector/issues/125)

### CODE REVIEW

File: DiverseSelector/distance.py [hunk 1]
```diff
@@ -220,21 +206,15 @@ def nearest_average_tanimoto(x: np.ndarray) -> float:
...
-            if dist < short and idx != jdx:
-                short = dist
-                a = idx
-                b = jdx
-        # calculate tanimoto for each shortest dist pair
-        tani.append(tanimoto(x[a], x[b]))
-    # compute average of all shortest tanimoto coeffs
+    # calculate euclidean distance between all points
+    #     and adjust for distance to self
+    dist = distance_matrix(x, x) + 100*np.eye(x.shape[0])
```
Comment (FarnazH) [inline DiverseSelector/distance.py:211]: @AWBroscius, we talked about changing `100` to `np.inf`. Can you please fix this?

File: DiverseSelector/distance.py [hunk 2]
```diff
@@ -220,21 +206,15 @@ def nearest_average_tanimoto(x: np.ndarray) -> float:
...
+    # find closest point for each row of x
+    short_idx = np.argmin(dist, axis=0)
+    print(f"these are the shortest indices:", short_idx)
```
Comment (FarnazH) [inline DiverseSelector/distance.py:214]: Remove the print statement.

File: DiverseSelector/distance.py [hunk 3]
```diff
@@ -220,21 +206,15 @@ def nearest_average_tanimoto(x: np.ndarray) -> float:
...
+    for idx in range(0, len(short_idx)):
```
Comment (FarnazH) [inline DiverseSelector/distance.py:215]: Better to use `for idx, min_d in enumerate(short_idx):` and change the body of for loop accordingly.

**Review by FarnazH — CHANGES_REQUESTED** [review]

**Review by FanwangM — APPROVED** [review]

**Comment by FanwangM** [comment]
The failing tests has been fixed by rebasing and I will address the styling in another PR soon.

**Comment by FarnazH** [comment]
I leave a note here, as this PR is already merged. Before #125 can be closed:

- The `modified_tanimoto` still needs to be finalized by @FarnazH.
- The module name should be changed as it contains similarity measures, not distances.

**Pull request merged**